src/DotNetty.Transport/Channels/IChannelHandler.cs

```diff
@@ -89,6 +89,7 @@ public interface IChannelHandler
         [PipelinePropagation(PropagationDirections.Inbound)]
         void ExceptionCaught(IChannelHandlerContext context, Exception exception);
 
+        [PipelinePropagation(PropagationDirections.Outbound)]
```

Comment:
One nitpick - the `PipelinePropagation` and `Skip` attributes need some XML-DOC comments that explain what they do inside PipelinePropagationAttribute.cs and SkipAttribute.cs respectively. Not immediately clear to me :p